### PR TITLE
Pull request for reijnaers2021re

### PR DESCRIPTION
**Pull request by damiaanr**
All figures and tables are organized in separate folders. We managed to get most of the document within the margins, except for table 3 and the appendix. I don't see how figure 5 could fit in the margins without a big loss in visibility. It would generally be possible to shrink figures 6-7. Table 3 would become a challenge to shrink without a significant loss in readability/relevant data.

Please note that the document does not compile on "standard" LaTeX environments, since it requires a relatively high amount of memory to render all figures. In my local environment I'm compiling on increased memory. It should be possible within any standard TeX environment to render a compressed version of the images (which are the ones used in the document on OpenReview). In journal/context.tex I added instructions in the comments.

### CODE REVIEW

File: reijnaers2021re/openreview/packages.tex [hunk 1]
```diff
@@ -0,0 +1,30 @@
+%\usepackage[utf8]{inputenc}
+
+\usepackage{biblatex}
+\addbibresource{bibliography.bib}
+
+\usepackage[T1]{fontenc}
```
Comment (koustuvsinha) [inline reijnaers2021re/openreview/packages.tex:6]: Please remove the import to `fontenc` as it breaks the ReScience fonts

File: reijnaers2021re/openreview/packages.tex [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+\usepackage{acronym}
+\usepackage{caption}
+\usepackage{subcaption}
+\usepackage{bbold}
+\usepackage{changepage}
+\usepackage{adjustbox}
+\usepackage{appendix}
+\usepackage{eurosym}
+\usepackage{standalone}
+
```
Comment (koustuvsinha) [inline reijnaers2021re/openreview/packages.tex:28]: You use `subfloat` in the content, which needs package `subfig` to be imported here:

```
\usepackage{subfig}
```

Comment (koustuvsinha) [reply reijnaers2021re/openreview/packages.tex:28]: Also remove the subcaption package while you are at it (will not work with subfig)

**Comment by koustuvsinha** [comment]
Thanks for adding the comments in the journal section, it was helpful! I think for our purposes since it's not optimal to have a separate build process for a single project, you can replace the `original/` with `compressed/`. (I was able to build with `compressed/` on my end, and the images in the appendix look good.

One suggestion I would have to tackle both Table 3 and Figure 5. Can you transpose them both? Since there are 7 models and 5 datasets, transposing should solve the issue. However, if transposing doesn't seem to be a viable option, can you break Table 3 into two parts, one containing 4 models and another 3? I agree by trying to fit the current table in the width would render all numbers unreadable.

File: reijnaers2021re/openreview/packages.tex (hunk 1 above)
Comment (damiaanr) [reply reijnaers2021re/openreview/packages.tex]: I don't understand how I could have missed this. Fixed!

File: reijnaers2021re/openreview/packages.tex (hunk 2 above)
Comment (damiaanr) [reply reijnaers2021re/openreview/packages.tex:27]: Thank you for spotting this. Fixed!

**Comment by damiaanr** [comment]
> Thanks for adding the comments in the journal section, it was helpful! I think for our purposes since it's not optimal to have a separate build process for a single project, you can replace the `original/` with `compressed/`. (I was able to build with `compressed/` on my end, and the images in the appendix look good.
> 
> One suggestion I would have to tackle both Table 3 and Figure 5. Can you transpose them both? Since there are 7 models and 5 datasets, transposing should solve the issue. However, if transposing doesn't seem to be a viable option, can you break Table 3 into two parts, one containing 4 models and another 3? I agree by trying to fit the current table in the width would render all numbers unreadable.

In the latest commit, I have tackled all these issues. I have confirmed alignment with the _showframe_ package. All elements should be within margins now, and, in my opinion, readable.

**Comment by koustuvsinha** [comment]
LGTM. In the last commit you have removed the symlinks to the font files, which breaks compilation. Feel free to add them back, or I can also do it post merge.

```
ln -s ../../template/roboto .
ln -s ../../template/source-code-pro .
ln -s ../../template/source-sans-pro .
ln -s ../../template/source-serif-pro .
```

**Comment by koustuvsinha** [comment]
@dodgejesse can you review the contents of the pdf? Thanks!

**Comment by damiaanr** [comment]
> LGTM. In the last commit you have removed the symlinks to the font files, which breaks compilation. Feel free to add them back, or I can also do it post merge.
> 
> From reijnaers2021re/journal, run these:
> 
> ```
> ln -s ../../template/roboto .
> ln -s ../../template/source-code-pro .
> ln -s ../../template/source-sans-pro .
> ln -s ../../template/source-serif-pro .
> ```

Sorry for that. They should be back on now.

**Review by dodgejesse — APPROVED** [review]
Looks good to me!

**Pull request merged**